amqp_broker.go

```diff
@@ -71,17 +71,23 @@ func NewAMQPConnection(host string) (*amqp.Connection, *amqp.Channel) {
 }
 
 // NewAMQPCeleryBroker creates new AMQPCeleryBroker
-func NewAMQPCeleryBroker(host string) *AMQPCeleryBroker {
-	return NewAMQPCeleryBrokerByConnAndChannel(NewAMQPConnection(host))
+func NewAMQPCeleryBroker(host string, queue ...string) *AMQPCeleryBroker {
+	queueName := DefaultQueueName
+	if len(queue) > 0 && queue[0] != "" {
```

Comment:
why do you get multiple queue names as argument in the first place?

Reply:
Answered in first comment, variadic is for optional parameters, only the first one is used.